cfgov/unprocessed/apps/youth-employment-success/js/views/route-details.js

```diff
@@ -250,17 +266,17 @@ function routeDetailsView( element, { alertTarget, hasDefaultTodo = false } ) {
       const costEstimate = getCalculationFn( route );
       const remainingBudget = money.subtract( budget.earned, budget.spent );
       const nextRemainingBudget = updateRemainingBudget(
-        remainingBudget,
+        String(remainingBudget),
```

Comment:
This is likely getting removed in favor of having the `money.js` helpers return strings after calculating, but I need to run the specs to make sure I'm not breaking things first 😄